kyaml/yaml/types_test.go

```diff
@@ -166,3 +167,86 @@ type: string
 	}
 	assert.Equal(t, expected, actual)
 }
+
+func TestIsMissingOrNull(t *testing.T) {
+	if !IsMissingOrNull(nil) {
+		t.Fatalf("input: nil")
+	}
+	// missing value or null value
+	node := &RNode{value: nil}
+	if !IsMissingOrNull(node) {
+		t.Fatalf("input: nil value")
+	}
+
+	node.value = &yaml.Node{}
+	if IsMissingOrNull(node) {
+		t.Fatalf("input: valid node")
+	}
+	// node with NullNodeTag
+	node.value.Tag = NullNodeTag
+	if !IsMissingOrNull(node) {
+		t.Fatalf("input: with NullNodeTag")
+	}
+
+	node.value = &yaml.Node{}
```

Comment:
Since this is a white box test, this section is redundant to L181.

Can you try some public methods here too please,e.g.
```
if IsMissingOrNull(NewScalarRNode("foo")) {
  error
}
```